add-kml-layer-with-network-links/README.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Layers",
+    "description": "Display a file with a KML network link, including displaying any network link control messages at launch.",
+    "formal_name": "AddKmlLayerWithNetworkLinks",
+    "ignore": false,
+    "images": [
+        "add-kml-layer-with-network-links.png"
+    ],
+    "keywords": [
+        "KML",
+        "KMZ",
+        "OGC",
+        "keyhole",
+        "network link",
+        "network link control",
+        "KmlDataset",
+        "KmlLayer"
+    ],
+    "language": "kotlin",
+    "redirect_from": "TODO",
```

Comment:
```suggestion
    "redirect_from": "",
```

Reply:
We're not redirecting from another sample page (which might happen if it's been renamed)